lib/osf-components/addon/components/file-renderer/component.ts

```diff
@@ -73,8 +74,8 @@ export default class FileRenderer extends Component {
     }
 
     didReceiveAttrs(): void {
+        this.isLoading = true;
         if (this.download !== this.lastDownload) {
```

Comment:
Does this do anything useful if the `this.isLoading = true` line is outside of it? I don't think we really care about this.lastDownload otherwise. We might be able to change `didReceiveAttrs()` to just setting `this.isLoading`. Thoughts?

Reply:
PR updated. I thing we used to care about it for some other purposes. Can't find any references to it anymore so I removed it.